### ISSUE TITLE
Invalid schema for DeploymentManifest.XSD

### ISSUE THREAD

**Issue by thespooler** [SharePoint/sp-dev-docs#4141]
DeploymentManifest.XSD is not valid as published here. The following elements refers to undefined types.
				&lt;xs:element name="UserX" type="DeploymentUserX" /&gt;
				&lt;xs:element name="GroupX" type="DeploymentGroupX" /&gt;
				&lt;xs:element name="RoleX" type="DeploymentRoleX" /&gt;
				&lt;xs:element name="RoleAssignmentX" type="DeploymentRoleAssignmentX" /&gt;

Also, it would be nice if those XSD could be provided as files, not as pastes from DOCX/PDF.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3b3bfe80-38e3-e49a-dd94-c2851264dfce
* Version Independent ID: 8796c402-cd21-a86b-99f0-d0bd035ceb5f
* Content: [SharePoint Online Migration API Overview](https://docs.microsoft.com/en-us/sharepoint/dev/apis/migration-api-overview#feedback)
* Content Source: [docs/apis/migration-api-overview.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/apis/migration-api-overview.md)
* Product: **sharepoint**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by markleavesley** [comment]
I agree it would be nice to download all the XSD files. If you follow the links through to the 2013 list, you have to follow more links to elements of the XSD files and make the file yourself, you cannot just download the XSD files.

**Comment by ashleywb** [comment]
Any news on this?

**Comment by netclectic** [comment]
Still no XSD files available to download :-/
